tests/app/confirmation_page/test_confirmation_page.py

```diff
@@ -15,7 +15,7 @@ def test_get_next_location_confirmation(self):
         answer_store = AnswerStore()
         answer_store.add(answer)
 
-        navigator = Navigator(load_schema_file("0_rogue_one.json"), answer_store)
+        navigator = Navigator(load_schema_file("0_rogue_one.json"), {}, answer_store)
```

Comment:
Nope, I disagree :) Navigator should work with both an empty AnswerStore and metadata. It's valid for me to request a path without metadata, which would be informed only by what's in the json. Similarly, it's valid for me to request a path based on answer store and the survey only...